README.md

```diff
@@ -3,7 +3,17 @@ This repository contains the code of the Azure IoT Identity Service and related
 
 # Installing
 
-TODO: List package names for each distro here.
+## CentOS
+
+```sh
+yum install aziot-identity-service
+```
+
+## Debian, Ubuntu
+
+```sh
+apt install aziot-identity-service
+```
```

Comment:
Both these commands probably need sudo, but that should be obvious

Reply:
I think I'll just remove this section. There's a lot more that goes into installing and configuring them than just the distro package, and we'll have docs.microsoft.com for that.

Reply:
Removed.